template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/EnableAuthenticator.cshtml

```diff
@@ -16,23 +15,25 @@
             <a href="https://go.microsoft.com/fwlink/?Linkid=825072">Android</a> and
             <a href="https://go.microsoft.com/fwlink/?Linkid=825073">iOS</a> or
             Google Authenticator for
-            <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en">Android</a> and
+            <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&amp;hl=en">Android</a> and
             <a href="https://itunes.apple.com/us/app/google-authenticator/id388497605?mt=8">iOS</a>.
         </li>
-        <li>Scan the QR Code or enter this key <kbd>@Model.PublicKey</kbd> into your two factor authenticator app. Spaces and casing do not matter.</li>
+        <li>Scan the QR Code or enter this key <kbd>@Model.Input.PublicKey</kbd> into your two factor authenticator app. Spaces and casing do not matter.</li>
         <li>
             Once you have scanned the QR code or input the key above your two factor authentication app will provide you
             with a unique code. Enter the code in the confirmation box below.
 
             <div class="row">
                 <div class="col-md-6">
                     <form method="post">
+                        <input asp-for="Input.PublicKey" type="hidden" />
```

Comment:
To round trip the public key so that if the user enters the wrong code on POST it shows up again.